src/caselawclient/models/utilities/dates.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from dateutil.parser import isoparse
+from pytz import UTC, tzinfo
+
+
+def parse_string_date(iso_string: str, timezone: tzinfo.BaseTzInfo) -> datetime:
```

Comment:
Is it worth naming this something like `parse_date_string_and_add_timezone`? Not totally sold on that exact wording, just a bit more explicit.

Reply:
gone with parse_date_string_as_utc